modules/Bio/EnsEMBL/Compara/Utils/JIRA.pm

```diff
@@ -556,20 +654,17 @@ sub _post_request {
     my $response = $agent->request($request);
     # Check and report possible errors
     if ($response->code() == 401) {
-        $self->{_logger}->error(
-            'Incorrect JIRA password. Please, try again.', 0, 0);
```

Comment:
I think this looks nicer, but I think following the blame rules, it should be left as is? Or am I missing something?

Reply:
I changed it because I'm the previous one to blame for this bit, so I'm not increasing my guilt :sweat_smile: